JenkinsfileStageTrigger

```diff
@@ -0,0 +1,234 @@
+/**
+ * CI Stage Pipeline Trigger
+ *
+ * This is a declarative pipeline for the CI stage pipeline
+ * that includes the building of images based on PRs
+ *
+ */
+
+// Openshift project
+openshiftProject = "continuous-infra"
+DOCKER_REPO_URL = '172.30.254.79:5000'
+
+// Defaults for SCM operations
+env.ghprbGhRepository = env.ghprbGhRepository ?: 'CentOS-PaaS-SIG/upstream-fedora-pipeline'
+env.ghprbActualCommit = env.ghprbActualCommit ?: 'master'
+
+// If this PR does not include an image change, then use this tag
+STABLE_LABEL = "stable"
+tagMap = [:]
+
+// Initialize
+tagMap['jenkins-continuous-infra-slave'] = STABLE_LABEL
```

Comment:
So we only need labels for the containers we use. Furthermore, we only need labels for the containers that are maintained in this repo. So realistically, you can just use STABLE across the board.